syntax = "proto3";

package translate;

service TranslationService {
  // Client sends a message
  rpc SendAudioBytes (AudioBytes) returns (Empty);

  // Server streams messages to client
  rpc ReceiveTextResponse (Empty) returns (stream TextResponse);
}

message AudioBytes {
    bytes audioBytes = 1;
}

message TextResponse {
    string text = 1;
}

message Empty {}

